configure.ac

```diff
@@ -19,6 +19,19 @@ AC_CONFIG_SRCDIR(src)
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
 
+dnl Make sure the pkg-config macros are defined
+m4_ifdef([PKG_PROG_PKG_CONFIG],[],[
```

Comment:
btw, maybe it is better to m4_include(m4/pkg.m4) as part of libvma and ensure it is available?